Fix a FTBFS with libpoppler 0.11: the destructor of GfxFont is virtual, and it doesn't make sense to call it.
Index: texlive-bin-2007.dfsg.2/build/source/texk/web2c/pdftexdir/pdftoepdf.cc
===================================================================
--- texlive-bin-2007.dfsg.2.orig/build/source/texk/web2c/pdftexdir/pdftoepdf.cc	2009-09-28 08:44:25.000000000 +0000
+++ texlive-bin-2007.dfsg.2/build/source/texk/web2c/pdftexdir/pdftoepdf.cc	2009-09-28 09:25:56.000000000 +0000
@@ -633,7 +633,7 @@
     }
     for (r = encodingList; r != 0; r = n) {
         n = r->next;
-        delete r->font;
+//        delete r->font;
         delete r;
     }
 }
